apps/eew/scfinder/main.cpp

```diff
@@ -1126,29 +1191,52 @@ class App : public Client::StreamApplication {
 		Core::TimeSpan                 _finderProcessCallInterval;
 		Core::TimeSpan                 _finderScanCallInterval;
 		Core::TimeSpan                 _finderMaxEnvelopeBufferDelay;
+		Core::TimeSpan                 _finderClipTimeout;
 		bool                           _finderAmplitudesDirty;
 		bool                           _finderScanDataDirty;
 
 		LocationLookup                 _locationLookup;
 		Finder_List                    _finderList;
 		PGA_Data_List                  _latestMaxPGAs;
+		std::string                    _preferredGainUnits;
 };
 
 
-bool App::Buddy::updateMaximum(const Core::Time &minTime) {
+bool App::Buddy::updateMaximum(const Core::Time &minTime, const std::string preferredGainUnits) {
 	Amplitude lastMaximum = maxPGA;
 	maxPGA = Amplitude();
 
 	if ( !pgas.empty() && pgas.back().timestamp >= minTime ) {
 		// Update maxmimum
 		PGABuffer::iterator it;
 		for ( it = pgas.begin(); it != pgas.end(); ++it ) {
-			if ( it->timestamp < minTime ) continue;
```

Comment:
@jenrandrews @maboese: I think removing this was a big mistake of mine. I am fixing it now...